src/components/ButtonWithHeading/index.js

```diff
@@ -9,7 +9,7 @@ export const ButtonWithHeading = ({ heading, buttonText, onClick = () => {}}) =>
       <Container>
         <Row>
           <Col> 
-            <h1>{!heading || (heading)}</h1>
+            <h1>{ heading && (heading)}</h1>
```

Comment:
Just saw the codebase, ! is used everywhere. So you can revert this commit to maintain code consistency

Reply:
Which commit ?

Reply:
So, do I need to use ! again for null check ?

Reply:
Yeah

Reply:
Done.